hazelcast-sql/src/test/java/com/hazelcast/jet/sql/impl/type/BasicNestedFieldsTest.java

```diff
@@ -383,32 +429,32 @@ public void test_typeCoercionUpserts() {
     @Test
     public void test_compoundAliases() {
         initDefault();
-        assertRowsAnyOrder(testInstance(), "SELECT ((org).office).name FROM " +
-                        "(SELECT (this).organization as org FROM (SELECT * FROM test))",
+
+        assertRowsAnyOrder(testInstance(),
+                "SELECT ((org).office).name FROM (SELECT (this).organization as org FROM (SELECT * FROM test))",
                 rows(1, "office1"));
+
         assertRowsAnyOrder(testInstance(), "SELECT (((this).organization).office).name FROM (SELECT * FROM test)",
                 rows(1, "office1"));
     }
 
     @Test
     public void test_newDotOperatorSyntax() {
         initDefault();
+
         assertRowsAnyOrder(testInstance(), "SELECT (((this).organization).office).name FROM test",
                 rows(1, "office1"));
     }
 
     @Test
     public void test_joins() {
         initDefault();
-        execute("CREATE MAPPING test2 (__key BIGINT, this UserType) "
-                + "TYPE IMap OPTIONS ("
-                + "'keyFormat'='bigint', "
-                + "'valueFormat'='java', "
-                + "'valueJavaClass'='" + User.class.getName() + "')");
+        createJavaMapping("test2", User.class, "this UserType");
```

Comment:
I had to check `createJavaMapping` to understand this line, at first sight `User` should be the mapping key. Maybe rename the method to indicate that, eg. to `createJavaValueMapping`?

Reply:
To prevent such confusion, I moved `createJavaMapping()` before the `@Test`s. Additionally, when you hover the method name, parameter names are displayed.